smart_fact_crawler/__init__.py

```diff
@@ -258,13 +258,22 @@ def main_page(url=None):
     if url is None:
         url = os.path.join(smartfacturl, 'fact.data')
     table = smartfact2table(url)
+
```

Comment:
Do you think we should make this try catch block a method of the table returned by smartfact2table? So  that all table requests profit from it? Or might this even harm? Just saying because it looks odd that only MAGIC weather needs it.